packages/office-ui-fabric-react/src/components/Panel/Panel.tsx

```diff
@@ -175,9 +175,7 @@ export class Panel extends BaseComponent<IPanelProps, IPanelState> implements IP
                 ) }
               style={ customWidthStyles }
               elementToFocusOnDismiss={ elementToFocusOnDismiss }
-              isClickableOutsideFocusTrap={
-                isLightDismiss || isHiddenOnDismiss || (focusTrapZoneProps && focusTrapZoneProps.isClickableOutsideFocusTrap)
-              }
+              isClickableOutsideFocusTrap={ true }
```

Comment:
this seems like a sledgehammer solution. I'm not sure if this is correct. Perhaps someone from @antonlabunets team could help validate? (@zoarif or @srideshpande  maybe)

Reply:
I agree it does seem like a 'sledgehammer' solution.  Obviously, if your app needs to allow these clicks you can set the implementation to true and be good.  However, this will block the usage of Grammarly and any third party extensions and such completely since they will be outside the focus zone.

The main thing is it seems unnecessary since there is an overlay blocking the rest of the page while the panel is open with default settings.  The only area I am unsure about is when you navigate the page using something other than a mouse - I didn't have any obvious issues that way but I'm not an expert on accessibility testing.

I'm open to ideas.

Reply:
Instead of changing the existing logic, may be add a prop in Panel for isClickableOutsideFocusTrap and set to true in your app?

Reply:
May be set the default to true and provide a prop if the app wants to set it to false?